tests/fixtures/addon/npm/package.json

```diff
@@ -21,7 +21,7 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.16.0"
+    "ember-cli-babel": "^7.1.2"
```

Comment:
missing comma

Reply:
@Turbo87 do we need comma here? It's the only entry in `dependencies` section